src/test/java/freemarker/manual/CustomFormatsExample.java

```diff
@@ -49,7 +50,9 @@ public void aliases1() throws IOException, TemplateException {
 
         Map<String, TemplateDateFormatFactory> customDateFormats
                 = new HashMap<String, TemplateDateFormatFactory>();
-        customDateFormats.put("fileDate", new AliasTemplateDateFormatFactory("dd/MMM/yy hh:mm a"));
+        Map<Locale, String> m=new HashMap<Locale,String>();
```

Comment:
This change is should be unnecessary, because if you don't specify a locale `Map`, the specified default pattern will be used regardless of your locale. Does this test fail for you without the `Map`? That can be a symptom of some other bug then.